packages/@aws-cdk/aws-codedeploy/test/test.deployment-group.ts

```diff
@@ -0,0 +1,36 @@
+import { expect, haveResource } from '@aws-cdk/assert';
+import autoscaling = require('@aws-cdk/aws-autoscaling');
+import ec2 = require('@aws-cdk/aws-ec2');
+import cdk = require('@aws-cdk/cdk');
+import { Test } from 'nodeunit';
+import codedeploy = require('../lib');
+
+// tslint:disable:object-literal-key-quotes
+
+export = {
+    'CodeDeploy Deployment Group': {
+        "created with ASGs contains the ASG names"(test: Test) {
+            const stack = new cdk.Stack();
+
+            const asg = new autoscaling.AutoScalingGroup(stack, 'ASG', {
+                instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.Standard3, ec2.InstanceSize.Small),
+                machineImage: new ec2.AmazonLinuxImage(),
+                vpc: new ec2.VpcNetwork(stack, 'VPC'),
+            });
+
+            new codedeploy.ServerDeploymentGroup(stack, 'DeploymentGroup', {
+                autoScalingGroups: [asg],
+            });
```

Comment:
I think `deploymentGroup.addAutoScalingGroup(asg)` will be a nice addition to the API

Reply:
Sure.